config/routes.rb

```diff
@@ -7,7 +7,14 @@
   # root "articles#index"
 
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :show, :new, :create]
   end
+
+  
+  # Add comment to a post
+  post 'users/:user_id/posts/:id/', to: 'posts#comment_post', as: 'comment_post'
+
```

Comment:
- [ ] Please create comments controller with `new` and `create` methods and remember to work on the `comment` form as well